bundles/org.openhab.binding.solax/README.md

```diff
@@ -113,6 +114,47 @@ If you're missing a channel this means that it's not supported for your inverter
+| charger-output-power-phase3              | Number:Power                | Power to/from the charger phase 3. [W]                        |
+| charger-total-output-power               | Number:Power                | Power from the charger on all phases. [W]                     |
+| charger-current-phase1                   | Number:ElectricCurrent      | Current from the charger phase 1. [A]                         |
+| charger-current-phase2                   | Number:ElectricCurrent      | Current from the charger phase 2. [A]                         |
+| charger-current-phase3                   | Number:ElectricCurrent      | Current from the charger phase 3. [A]                         |
+| charger-voltage-phase1                   | Number:ElectricPotential    | Voltage of the charger's phase 1. [V]                         |
+| charger-voltage-phase2                   | Number:ElectricPotential    | Voltage of the charger's phase 2. [V]                         |
+| charger-voltage-phase3                   | Number:ElectricPotential    | Voltage of the charger's phase 3. [V]                         |
+| charger-external-current-phase1          | Number:ElectricCurrent      | Current from the provider phase 1. [A]                        |
+| charger-external-current-phase2          | Number:ElectricCurrent      | Current from the provider phase 2. [A]                        |
+| charger-external-current-phase3          | Number:ElectricCurrent      | Current from the provider phase 3. [A]                        |
+| charger-external-power-phase1            | Number:Power                | Power from the provider phase 1. [W]                          |
+| charger-external-power-phase2            | Number:Power                | Power from the provider phase 2. [W]                          |
+| charger-external-power-phase3            | Number:Power                | Power from the provider phase 3. [W]                          |
+| charger-external-total-power             | Number:Power                | Total power from the provider. [W]                            |
+| charger-plug-temperature                 | Number:Temperature          | Temperature of the charger's plug. [°C]                       |
+| charger-internal-temperature             | Number:Temperature          | Internal temperature on the board of the charger. [°C]        |
+| charger-mode                             | String                      | Charger Workmode.                                             |
+| charger-state                            | String                      | Charger State.                                                |
+| last-update-time                         | DateTime                    | Last time with a successful retrieval of data.                |
+| raw-data                                 | String                      | The raw JSON data retrieved from the inverter's Wi-Fi module. |
```

Comment:
Don't expose raw channels. Data from the remote service should be mapped to the openHAB domain.

Reply:
I disagree. Raw data gives you the flexibility if they introduce new channels in the API to use them directly (via JSONPath or by some other mapping transformation)

Reply:
I understand that use case, it comes with pros and cons. 

There has been a discussion about such a case before and it was then decided to not offer the raw channel.

Reply:
OK. I will remove it. What are the cons? The persistency layer?

Reply:
Persistence indeed. But there is more, can't remember all the details but one of the biggest pro is also the biggest con. openHAB has an architecture with Things, Channels, Items etc. Exposting raw channels undermines that architecture. It's noy only a technical limitation, but also long term user experience, maintainability etc.

Reply:
OK. Should I work in direction of decommissioning the raw data channels from the other things in this binding then? Probably I can put a note in the community thread of the binding if that's the general aim of the core developers...

Reply:
As they are already there it’s breaking to remove them. Maybe mark them as advanced and or obsolete if not already.

Reply:
They're already advanced.